admin.rst

```diff
@@ -39,6 +39,7 @@ server-host       String  \*4
 server-port       Int     3000
 server-proxy-url  String
 jwt-secret        String
+secret-is-base64  Bool    False    N
```

Comment:
The other optional parameters leave the required column blank rather than using the letter `N`. Either remove this one, or add `N` to the other optional params.